.github/workflows/release.yml

```diff
@@ -140,9 +140,8 @@ jobs:
         id: commit_docs
         uses: stefanzweifel/git-auto-commit-action@v4
         with:
-          commit_message: "[issue#${{ github.event.inputs.issue_number }}] Update Helm repo index"
+          commit_message: "Update Helm repo index"
           commit_options: --amend --signoff
```

Comment:
`--amend` will add your changes to the existing commit. Please remove it.

Reply:
Done